registry/api/v2/urls.go

```diff
@@ -174,15 +174,23 @@ func (ub *URLBuilder) BuildBlobUploadURL(name reference.Named, values ...url.Val
 // including any url values. This should generally not be used by clients, as
 // this url is provided by server implementations during the blob upload
 // process.
-func (ub *URLBuilder) BuildBlobUploadChunkURL(name reference.Named, uuid string, values ...url.Values) (string, error) {
+func (ub *URLBuilder) BuildBlobUploadChunkURL(name reference.Named, uuid string, relative bool, values ...url.Values) (string, error) {
```

Comment:
Stylistically, I have a preference for making the relative version a different function, since there is only one line of code in common.